### ISSUE TITLE
Multiple file upload

### ISSUE THREAD

**Issue by merforga** [GW2Raidar/gw2raidar#54]
Have encountered multiple times now when uploading batch files (again generally around the 8 or 9+ files mark. Uploaded with following scenarios:
- Bulk drag and drop
- Drag and drop individually in quick succession

Performance issues aside in terms of length of time to analyse (some covered under #47 and #39), if left to complete, it will stall and not analyse / provide updates for the last file uploaded (I think it's the last file but it's always the last one remaining to be analysed). 

Basically, as an example for 9 files. 
Upload will complete all files, 9 in blue circle indicator
As they're uploaded and being processed it'll show 9 orange
When they finished it'll be like 8 orange, 1 green > 7 orange, 2 green etc etc
When it gets to 1 orange, 8 green, it just stalls indefinitely (left it for over an hour with no result.
